src/worker.ts

```diff
@@ -13,6 +14,17 @@ interface Worker {
 const cloudflareWorker: Worker = {
   fetch: async (request, env, ctx) => {
     const cache = caches.default;
+
+    const sentry = new Toucan({
```

Comment:
How much performance this impacts for each request? 👀

Reply:
There really shouldn't be any measurable impact, we probably could move the construction of it inside the catch scope but put it there just in case in the future if we ever need it in some other scope.